pipeline/train/spm-vocab.sh

```diff
@@ -65,6 +65,12 @@ fi
 
 vocab_dir=$(dirname "${vocab_src_output}")
 mkdir -p "${vocab_dir}"
+user_symbols="
+__source__,__target__,__done__,__start__,__end__,__sep__,
+__misc0__,__misc1__,__misc2__,__misc3__,__misc4__,
+__misc5__,__misc6__,__misc7__,__misc8__,__misc9__,
+"
```

Comment:
@ZJaume, one question here: did you add \n on purpose? Are there any negative implications for the current models from this? I assume the models will never encounter it in our current data.